Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -365,9 +372,12 @@ instructionLabel(_rethrow)
     jumpToException()
 
 instructionLabel(_throw_ref)
-    popQuad(a2, t1)
+    popQuad(a2)
     bieq a2, ValueNull, .throw_null_ref
 
+if X86_64
+    loadp UnboxedWasmCalleeStackSlot[cfr], ws0
+end
```

Comment:
Why do we need to do this only for x64?

Reply:
Since x64 doesn't have as many registers, ws0 conflicts with t0 and r0, so we have to reload the callee in these situations. ARM doesn't have that register conflict, so it isn't necessary there.